framework/ui/qml/Muse/Ui/qmlaccessible.cpp

```
@@ -365,6 +369,48 @@ int AccessibleItem::accessibleCharacterCount() const
+    if (selectionIndex != 0 || !m_textItem) {
+        return;
+    }
+
+    const int count = accessibleCharacterCount();
+    const int start = std::clamp(startOffset, 0, count);
+    const int end = std::clamp(endOffset, 0, count);
+
+    //! NOTE Accessibility calls may arrive on a thread other than the GUI one
+    //! (in-process UIA providers can be invoked from an RPC thread), so the write
+    //! is queued onto the text item's own thread.
+    QQuickItem* textItem = m_textItem;
```

Clang (14.0.6)

[warning] variable 'textItem' is not initialized

(cppcoreguidelines-init-variables)